### ISSUE TITLE
setSkipFirstHeader not working

### ISSUE THREAD

**Issue by kolaval** [LibrePDF/OpenPDF#536]
**Describe the bug**
Calling table.setSkipFirstHeader(true) does not skip the header rows on the first page.

**To Reproduce**
create basic table that spans multiple pages.
```
document.open();
PdfPCell cell;
PdfPTable mainTable = new PdfPTable(1);
mainTable.setWidthPercentage(100);

cell = new PdfPCell(new Phrase("HEADER 1"));
cell.setBorder(Rectangle.BOX);
mainTable.addCell(cell);

cell = new PdfPCell(new Phrase("HEADER 2"));
cell.setBorder(Rectangle.BOX);
mainTable.addCell(cell);

cell = new PdfPCell(new Phrase("HEADER 3"));
cell.setBorder(Rectangle.BOX);
mainTable.addCell(cell);

for (int i = 0; i < 100; i++) {
	cell = new PdfPCell(new Phrase("ROW "+i));
	cell.setBorder(Rectangle.BOX);
	mainTable.addCell(cell);
}

mainTable.setHeaderRows(3);
mainTable.setSkipFirstHeader(true);
mainTable.setSplitLate(false);
document.add(mainTable);
```

**Expected behavior**
First page should not have HEADER 1,2,3 rows.

Screenshots
![image](https://user-images.githubusercontent.com/4124487/117771631-943dfa80-b236-11eb-8c35-61baf02dea6c.png)

**Comment by Wugengxian** [comment]
@kolaval I have fixed it #537, it is a little problem.

**Comment by kolaval** [comment]
@asturio Issue again in 1.3.28
